.github/conda_pgm_env.yml

```diff
@@ -7,8 +7,10 @@ dependencies:
   - wheel
   - setuptools
   - boost-cpp
+  - nlohmann_json
+  - msgpack-cxx
   - eigen
   - numpy
   - pytest
   - pytest-cov
-  - msgpack-cxx
+  - msgpack
```

Comment:
`msgpack-cxx` is the correct name, you don't need `msgpack`

Reply:
i believe we need both because the `python` tests make use of `msgpack` as well

Reply:
i see what i did wrong. should be `msgpack-python` for the unit tests